Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm

```diff
@@ -40,11 +40,11 @@
 #import "ChromeClient.h"
 #import "FontCascade.h"
 #import "FrameSelection.h"
-#import "HitTestResult.h"
 #import "HTMLFrameOwnerElement.h"
 #import "HTMLInputElement.h"
 #import "HTMLNames.h"
 #import "HTMLTextAreaElement.h"
+#import "HitTestResult.h"
```

Comment:
Header sort must be case-sensitive (as `sort` would produce) – code style is explicit on that 🤷.